sql/moz-fx-data-shared-prod/fenix_derived/new_profile_activation_v1/metadata.yaml

```diff
@@ -24,8 +24,5 @@ scheduling:
     - 'search_derived'
     - 'mobile_search_clients_daily_v1'
   - - 'moz-fx-data-shared-prod'
-    - 'fenix'
-    - 'baseline_clients_first_seen'
-  - - 'moz-fx-data-shared-prod'
-    - 'fenix'
-    - 'baseline_clients_last_seen'
+    - 'org_mozilla_firefox_derived'
+    - 'baseline_clients_last_seen_v1'
```

Comment:
`baseline_clients_last_seen_v1` is dependent on `baseline_clients_first_seen_v1`, so I removed the latter from the explicit dependencies here.